sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -425,8 +427,8 @@ async def update_queue(
         :type queue: ~azure.servicebus.management.QueueProperties
```

Comment:
don't know if this is needed, but maybe you can add to the docstring that these methods accept keyword args to override properties? like:
```suggestion
        :type queue: ~azure.servicebus.management.QueueProperties
        :keyword datetime.timedelta auto_delete_on_idle: ....
        ...
```
or a link to all the keyword args? (same comment applies to all update_*)

Reply:
I have added a line saying that user could pass keyword argument to update properties.